### ISSUE TITLE
change lambda at each iteration in traveltime inversion

### ISSUE THREAD

**Issue by makeabhishek** [gimli-org/gimli#625]
Is it possible to change the lambda value in each iteration in inversion rather than keeping it constant?

**Comment by carsten-forty2** [comment]
Yes. [Inversion.run](https://www.pygimli.org/pygimliapi/_generated/pygimli.frameworks.html#pygimli.frameworks.Inversion.run) accepts the keyword `lambdaFactor`. If you use `invert(..., **kwargs)` from any Manager then typically all meaningfull `kwargs` argument are forwarded to the appropriate `inversion.run` call.

**Comment by makeabhishek** [comment]
Thanks for pointing this. Is the `lambdaFactor` is a multiplication factor to `lambda`.
